modules/enabling-multi-cluster-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * assemblies/web-console.adoc
+
+:_content-type: PROCEDURE
+[id="enable-multi-cluster-console_{context}"]
+= Enabling multicluster in the web console
+
+:FeatureName: Multicluster console
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+.Prerequisites
+* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management (ACM) for Kubernetes 2.5] installed.
+* You must have upgraded to the latest version of {product-title}.
+* You must have administrator privileges.
+
+WARNING: This should not be set on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it cannot be undone.
+
+.Procedure
+
+. Log in to the {product-title} web console using your credentials.
+
+. Enable ACM in the administrator perspective by following *Admin* -> *ClusterSettings* -> *Configuration* -> *Console Details* -> *Console Plugins* and click `enable`.
+
+. Click *Refresh the web console* in the pop-up window that states that a web console update is available.
+
+. *local-cluster* and *All Clusters* is visible above the perspectives in the navigation section.
+
+. Enable the feature gate from *Administration* -> *Cluster Settings* -> *Configuration* -> *FeatureGate*, and edit the YAML.
++
+[source,yaml]
+
+----
+spec:
+    featureSet: TechPreviewNoUpgrade
+----
+
+. Click save. This enables for all clusters.
+IMPORTANT: Once you save, this cannot be undone.
```

Comment:
```suggestion
[IMPORTANT] 
====
After you save, this feature is enabled and cannot be undone.
====
```
Re: [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o) guidance on _once_ and pronoun usage.